parsons/freshdesk/freshdesk.py

```diff
@@ -213,3 +230,39 @@ def get_agents(self, email=None, mobile=None, phone=None, state=None):
         tbl.remove_column("signature")  # Removing since raw HTML might cause issues.
 
         return tbl
+
+    def create_ticket(
+        self, subject, description, email, priority, status, cc_emails=None, custom_fields=None
+    ):
+        """
+        Create a ticket in Freshdesk.
```

Comment:
Looks like cc_emails and custom_fields are optional, can you mark them as such in the docs with a "(optional)" next to the param name?

Reply:
Hi @shaunagm ofc!
done.
Thank you so much for reviewing and for the feedback!

Reply:
Thank you!